android/library/src/main/java/io/logto/android/authflow/browser/BrowserAuthFlowProperty.kt

```diff
@@ -0,0 +1,11 @@
+package io.logto.android.authflow.browser
+
+import io.logto.android.callback.AuthenticationCallback
+import io.logto.android.config.LogtoConfig
+
+data class BrowserAuthFlowProperty(
```

Comment:
`BrowserAuthFlowProperty` means nothing to me. maybe `AuthConfig` is enough? since it looks like we can share the same stuff with native flow as well

Reply:
good idea. i'll rename it to `BrowserAuthConfig` temporarily and rename it to `AuthConfig` the next time i need it